### PR TITLE
Ryans lab meeting functions: fano factor values

### PR DESCRIPTION
**Pull request by rcanfield2**

### CODE REVIEW

File: aopy/analysis.py [hunk 1]
```diff
@@ -292,3 +292,23 @@ def interpolate_extremum_poly2(extremum_idx, data, extrap_peaks=False):
 
     return extremum_time, extremum_value, f
 
+def get_fano_factor_values_per_condition(spiking_data):
```
Comment (rcanfield2) [inline aopy/analysis.py:295]: Make function more generic because this isn't actually calculating the fano factor

File: aopy/analysis.py [hunk 2]
```diff
@@ -292,3 +292,23 @@ def interpolate_extremum_poly2(extremum_idx, data, extrap_peaks=False):
...
+    '''
+    This function calculates the two parameters used to calculate fano factor for input spiking data based on Churchland et al. 2010.
```
Comment (rcanfield2) [inline aopy/analysis.py:297]: Put a doi to Churchland et al. 2010 to ensure the right one is found.

File: aopy/analysis.py [hunk 3]
```diff
@@ -292,3 +292,23 @@ def interpolate_extremum_poly2(extremum_idx, data, extrap_peaks=False):
...
+    These two parameters are calculated for each unit and are the mean spikes per trial and the spiking variance of each unit across trials.
+
```
Comment (rcanfield2) [inline aopy/analysis.py:299]: Show documentation about where this is used

File: aopy/analysis.py [hunk 4]
```diff
@@ -292,3 +292,23 @@ def interpolate_extremum_poly2(extremum_idx, data, extrap_peaks=False):
...
+    Args:
+        spiking_data (ntime, nunits, ntr): Input spiking data
+        weight_regression (bool): A flag to weight the linear regression based on the number of trials.
```
Comment (rcanfield2) [inline aopy/analysis.py:302]: remove this variable input.

File: aopy/analysis.py (hunk 3 above)
Comment (rcanfield2) [reply aopy/analysis.py:299]: Add assumptions made by function. If not here than up stream.

**Review by leoscholl — APPROVED** [review]

**Pull request merged**